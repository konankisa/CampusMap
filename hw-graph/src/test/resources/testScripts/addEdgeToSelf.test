# Creates an empty graph, adds a node to it,
# and adds an edge to itself

# Create a graph
CreateGraph graph

# Add a node
AddNode graph node

# Add an edge from node to node in graph
AddEdge graph node node edge