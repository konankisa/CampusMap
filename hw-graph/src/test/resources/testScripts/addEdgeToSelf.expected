# Creates an empty graph, adds a node to it,
# and adds an edge to itself

# Create a graph
created graph graph

# Add a node
added node node to graph

# Add an edge from node to node in graph
added edge edge from node to node in graph